pants.ini

```diff
@@ -263,6 +267,12 @@ skip: True
 [fmt.scalafix]
 skip: True
 
+[protoc.gen.go-protobuf]
+version=3.4.1
+
```

Comment:
Which of these tool definitions is actually being used? I think maybe the `gen.go-protobuf` one? You should be able to remove/replace the subsystem dependency on `Protoc` from either your subsystem or the task, depending on which is actually using it.

Reply:
If we use `options_scope = 'protoc_gen_go'` here's an example:

```
[root@ea70b2190df0 pants]# ./pants --help-all | grep -- --protoc | sort -u
--protoc-gen-go-protobuf-version=<str> (default: '3.4.1')
--protoc_gen_go-version=<str> (default: 'v1.1.0')
```

Notice how the prefix "protoc gen go" prefix is the same in both cases, with just the delimiter different due to the options_scope.

No proposed change at this time, just noticing something through the lens of a user. Really I noticed this while trying to update some docs and thinking, wow this is confusing :)